frontend/src/utils/showToastOnError.ts

```diff
@@ -0,0 +1,31 @@
+import type { ApiError } from "../client"
+
+interface ErrorDetail {
+  msg: string
+}
+
+interface ApiErrorBody {
+  detail?: string | ErrorDetail[]
+}
+
+// These variables are not used, they are just type definition
```

Comment:
you don't need this note

Reply:
It's a comment for why I'm disabling eslint